deploy/helm/opsbot/values.yaml

```diff
@@ -3,7 +3,7 @@ replicaCount: 1
 
 image:
   repository: maibornwolff/opsbot
-  tag: v0.0.1
+  tag: dev
```

Comment:
tag must be `v0.0.1` for the release pipeline to work

Reply:
Done!